README.md

```diff
@@ -15,11 +15,21 @@ cd ../drupal
 ln -s ../drupal-admin-ui/build vfancy
 
 ## Link the support module to Drupal
-cd ../drupal
-ln -s ../drupal-admin-ui/admin_ui_support modules
+cd ../drupal/modules
+ln -s ../../drupal-admin-ui/admin_ui_support .
+
+## Install the jsonapi module
```

Comment:
We should say "Download" here because we "Install" bellow.

Reply:
+1 Do you want to fix it?